int32 SHOULDER=0
int32 ELBOW=1
int32 WRIST=2
int32 GRIPPER=3
int32 BASE=4
int32 LIGHT=5

int32 STOP=0
int32 DOWN=1
int32 UP=2

int32 motorIdx
int32 motorState
